crates/aptos/src/common/types.rs

```diff
@@ -1657,6 +1657,11 @@ pub struct TransactionOptions {
     #[clap(flatten)]
     pub prompt_options: PromptOptions,
 
+    /// If true, skip simulation. If this is set, you must set --max-gas. Be careful,
+    /// setting this means you could submit a transaction that will fail.
+    #[clap(long)]
+    pub(crate) skip_simulation: bool,
```

Comment:
Functionally, this is the same as setting max-gas.  Since simulation is skipped if you provide max gas.

Reply:
Yeah you're right I don't think this PR is necessary.

>  I think the only difference here (which I didn't check), is you add simulation if you provide max gas and not the skip flag?

I didn't even do it that way, the PR as written is completely unnecessary.